src/plugins/saved_objects_management/public/management_section/objects_table/components/duplicate_modal.tsx

```diff
@@ -0,0 +1,467 @@
+        <b style={{ color: '#000' }}>not</b> be copied, because they have already existed in the
+        selected workspace or they are workspaces themselves.
+      </p>
+    );
+
+    const ignoreSomeObjectsChildren: React.ReactChild = (
+      <>
+        <EuiCallOut
+          title="Some saved objects will be ignored."
+          color="warning"
+          iconType="help"
+          aria-disabled={ignoredSelectedObjectsLength === 0}
+        >
+          {ignoredSelectedObjectsLength === 1
+            ? warningMessageForOnlyOneSavedObject
+            : warningMessageForMultipleSavedObjects}
+        </EuiCallOut>
+        <EuiSpacer />
+      </>
+    );
+
+    return (
+      <EuiModal
+        data-test-subj="savedObjectsDuplicateModal"
+        className="savedObjectsModal"
+        onClose={onClose}
+      >
+        <EuiModalHeader>
+          <EuiModalHeaderTitle>
+            <FormattedMessage
+              id="savedObjectsManagement.objectsTable.duplicateModal.title"
+              defaultMessage="Duplicate {duplicateMode, select, all {all objects} other {{objectCount, plural, =1 {{objectName}} other {# objects}}}}?"
+              values={{
+                duplicateMode,
+                objectName: allSelectedObjects[0].meta.title,
+                objectCount: allSelectedObjects.length,
+              }}
+            />
+          </EuiModalHeaderTitle>
+        </EuiModalHeader>
+
+        <EuiModalBody>
+          <EuiFormRow
+            fullWidth
+            label={
+              <FormattedMessage
+                id="savedObjectsManagement.objectsTable.duplicateModal.targetWorkspacelabel"
+                defaultMessage="Destination workspace"
+              />
+            }
+          >
+            <>
+              <EuiText size="s" color="subdued">
+                {'Specify a workspace where the objects will be duplicated.'}
+              </EuiText>
+              <EuiSpacer size="s" />
+              <EuiComboBox
+                options={workspaceOptions}
+                onChange={this.onTargetWorkspaceChange}
+                selectedOptions={targetWorkspaceOption}
+                singleSelection={{ asPlainText: true }}
+                onSearchChange={this.onSearchWorkspaceChange}
+                isClearable={false}
+                isInvalid={!confirmDuplicateButtonEnabled}
+              />
+            </>
+          </EuiFormRow>
+
+          <EuiSpacer size="m" />
+          {duplicateMode === DuplicateMode.All && this.renderDuplicateObjectCategories()}
+          {duplicateMode === DuplicateMode.All && <EuiSpacer size="m" />}
+
+          <EuiFormRow
+            fullWidth
+            label={
+              <FormattedMessage
+                id="savedObjectsManagement.objectsTable.duplicateModal.relatedObjects"
+                defaultMessage="Related Objects"
+              />
+            }
+          >
+            <>
+              <EuiText size="s" color="subdued">
+                {
```

Comment:
i18n

Reply:
Done

Reply:
Have modified it according to your suggestion.